### DISCUSSION TITLE
program fails to open due to removed BAAl entry on huggingface

### DISCUSSION THREAD

**Discussion by jgxxjg** [open-webui/open-webui#5243] [General]
OSError: BAAl/bge-reranker-v2-m3 is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'
Repository Not Found for url: https://huggingface.co/BAAl/bge-reranker-v2-m3/resolve/main/config.json

**Comment by cl93a** [comment]
I have a similar issue (below). Was this resolved (and how)?

2024-09-09 20:31:51 OSError: sentence-transformers/bge-m3 is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'
2024-09-09 20:31:51 If this is a private repository, make sure to pass a token having permission to this repo either by logging in with `huggingface-cli login` or by passing `token=<your_token>`

**Reply by jgxxjg** [reply]
I met the above error with docker install. But when I switched to the pip installation, everything works fine.

**Comment by pavluka6** [comment]
It seems like you have typo. You typed l as lowercase 'L', where BAAI should be written with uppercase 'i'

**Comment by flefevre** [comment]
I do not understand why this parameter is a model name.
For me it should be an url of openai compatible server such as litellm that would serve given the secret key a model for reranking.
So my 2 cent is to refactor the model parameter name into an url parameter + the secret key parameter
